geonode/upload/upload.py

```diff
@@ -920,12 +924,13 @@ def final_step(upload_session, user, charset="UTF-8", dataset_id=None):
                     with transaction.atomic():
                         resource_manager.set_permissions(
                             None, instance=saved_dataset, permissions=permissions, created=created)
+                        resource_manager.exec(
```

Comment:
Moved before the update, in this case the time_serie is already configured before sync the resource with geoserver